Dockerfile

```
@@ -1,4 +1,5 @@
-FROM node:18.16.1 as node
+ARG NODE_VERSION=18.16.1
+FROM node:"${NODE_VERSION}" as node
```

```suggestion
FROM node:18.16.1 as node
```

You might want to also remove this change (https://github.com/jenkins-infra/jenkins.io/pull/6455/commits/df679d551d19fcba141301408534da1a70b4c4a7) as a follow up of https://github.com/jenkins-infra/jenkins.io/pull/6455/commits/952acc0f2da29e8061d860e262b6577a73da3fd0 (ref. https://github.com/jenkins-infra/jenkins.io/pull/6455#issuecomment-1649748545) to keep this under the Renovabot update system